sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
+            applications.data ->> ''youthInCare'' as "Youth in Care Flag",
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
+            applications.data ->> ''outOfHighSchoolFor4Years'' as "Outside of highschool >4 years flag",
+            applications.data ->> ''whenDidYouGraduateOrLeaveHighSchool'' as "High School Graduation Date",
+            applications.data ->> ''fulltimelabourForce'' as "Recent Labour Force Participation Flag",
+            applications.data ->> ''addTrustContactToContactSABC'' as "Trusted Contact Identified (Yes/No)",
+            applications.data ->> ''relationshipStatus'' as "Marital Status",
+            --Application Financial Info
+            applications.data ->> ''taxReturnIncome'' as "Total income",
+            applications.data ->> ''partTimeAwardTypesToBeConsidered'' as "Grant-only flag",
+            applications.data ->> ''scholarshipsreceviedCosts'' as "Merit-based scholarship amount",
+            applications.data ->> ''parentvoluntarycontributionsCosts'' as "Voluntary parental contribution amount",
+            applications.data ->> ''governmentFundingCosts'' as "Government funding amount",
+            applications.data ->> ''nongovernmentfundingCosts'' as "Non-government funding amount",
+            applications.data ->> ''bcIncomeassistancefordisabilitiesCosts'' as "B.C. Income Assistance Amount",
+            applications.data ->> ''returntriphomeCost'' as "Relocation costs",
+            -- Application Financial Appeals Info
+            applications.data ->> ''exceptionalExpensesApplicationException'' as "Exceptional expense amount",
+            applications.data ->> ''daycareCosts11YearsOrUnder'' as "Daycare cost amount",
+            applications.data ->> ''totalUnsubsidizedDayCareCostsDuringClassHoursForTheProposedStudyPeriodForDisabledInfirmedDependents12YearsOfAgeAndOlderPanel'' as "Unsubsidized daycare amount",
+            ''--TBD--'' as "Child/Spousal Support amount",
+            ''--TBD--'' as "Living in parent house and paying living expenses amount",
+            applications.data ->> ''transportationcostsEstimate'' as "Additional transportation amount (as applicable)",
+            -- Parent Info Self-Reported by Student
+            applications.data ->> ''pleaseProvideAnEstimationOfYourParentsIncome'' as "Parental estimated income self-reported",
```

Comment:
I see we have the header saying `self-reported`, so do we need it to add `self-reported` to be added in all its fields? 
![image](https://github.com/bcgov/SIMS/assets/77353155/bbd037be-210f-4379-a66c-fc009951933d)

Reply:
I think so because in the csv it won't be shown if we remove the "self-reported".

Reply:
Oh is that what we want in CSV?Okay resolving